infrastructure/demo.tfvars

```diff
@@ -0,0 +1,7 @@
+idam_api_url = "https://idam-api.demo.platform.hmcts.net"
+
+idam_client_id = "adoption"
```

Comment:
aat.tfvars call it 
idam_client_id = "adoption-backend"
here it is 
idam_client_id = "adoption"

rathers should be same